diff --git a/node_modules/next/dist/server/dev/next-dev-server.js b/node_modules/next/dist/server/dev/next-dev-server.js
index 75be176..cf67e47 100644
--- a/node_modules/next/dist/server/dev/next-dev-server.js
+++ b/node_modules/next/dist/server/dev/next-dev-server.js
@@ -165,7 +165,7 @@ class DevServer extends _nextServer.default {
                             ...urlQuery,
                             ...query
                         };
-                        await this.render(req, res, page, mergedQuery, parsedUrl, true);
+                        await this.render(req, res, path, mergedQuery, parsedUrl, true);
                         return {
                             finished: true
                         };
