diff --git a/node_modules/next/dist/server/dev/next-dev-server.js b/node_modules/next/dist/server/dev/next-dev-server.js
index 913daf1..dd15c1d 100644
--- a/node_modules/next/dist/server/dev/next-dev-server.js
+++ b/node_modules/next/dist/server/dev/next-dev-server.js
@@ -137,7 +137,7 @@ class DevServer extends _nextServer.default {
                             ...urlQuery,
                             ...query
                         };
-                        await this.render(req, res, page, mergedQuery, parsedUrl, true);
+                        await this.render(req, res, path, mergedQuery, parsedUrl, true);
                         return {
                             finished: true
                         };
